### PR TITLE
Koyomi now respects the Calendar.firstWeekday property

### PR DESCRIPTION
**Pull request by EvilClay**

### CODE REVIEW

File: Koyomi/DateModel.swift [hunk 1]
```diff
@@ -80,6 +80,18 @@ final class DateModel: NSObject {
         return false
     }
     
+    func satudatIndex() -> Int {
+        return (7 - self.calendar.firstWeekday) % 7
```
Comment (shoheiyokoyama) [inline Koyomi/DateModel.swift]: saturdayIndex?

**Comment by EvilClay** [comment]
Yes, sorry.

**Comment by shoheiyokoyama** [comment]
@EvilClay Thank you for pull request.
I merged your PR :) Thank you!!

**Comment by shoheiyokoyama** [comment]
@EvilClay 
Your pull request inspired me to support international calendar, thank you!!!!
53c306a

**Pull request merged**